src/zenml/integrations/s3/__init__.py

```diff
@@ -30,7 +30,7 @@ class S3Integration(Integration):
     """Definition of S3 integration for ZenML."""
 
     NAME = S3
-    REQUIREMENTS = ["s3fs==2022.3.0"]
+    REQUIREMENTS = ["s3fs<=2022.11.0"]
```

Comment:
Can this also have a lower bound, e.g. bigger than the version we previously required? An open range is always dangerous, we might install some 2+ years old dependency like this